cloud-controller-manager/osc/tags.go

```diff
@@ -152,9 +154,10 @@ func (t *awsTagging) clusterTagKey() string {
 	return TagNameKubernetesClusterPrefix + t.ClusterID
 }
 
-func (t *awsTagging) hasClusterTag(tags []*ec2.Tag) bool {
+// To delete after last call to this function
+func (t *awsTagging) hasClusterAWSTag(tags []*ec2.Tag) bool {
 	debugPrintCallerFunctionName()
-	klog.V(10).Infof("hasClusterTag(%v)", tags)
+	klog.V(10).Infof("hasClusterAWSTag(%v)", tags)
```

Comment:
Osc Tag ?

Reply:
hasClusterAWSTag is still used, it will be removed when all calls are migrated